apps/ledger-live-mobile/src/screens/Onboarding/shared/infoPagesData.tsx

```diff
@@ -469,552 +461,3 @@ export const infoModalScenes = {
   hideRecoveryPhraseInfoModalProps,
   pairNewErrorInfoModalProps,
 };
-
-const pinCodeScenes = (
```

Comment:
Great 🧨

Reply:
kill it with fire
also this lottieAnimations at the top is only used in a debug screen so I think there's something weird there